data/core/commands/core.lua

```diff
@@ -86,10 +86,18 @@ command.add(nil, {
   end,
 
   ["core:open-file"] = function()
+    local suggestion = nil
+    if core.active_view ~= nil and core.active_view.doc ~= nil and core.active_view.doc.filename ~= nil then
```

Comment:
ok but in lua you don't need to write "~= nil" you can just write the expression itself. If it is nil it will evaluate to false but this is just a matter of style. You can leave as this if you want.
You may also hoist core.active_view at the beginning like:
local view = core.active_view